episodes/conditionals.md

```diff
@@ -296,7 +296,7 @@ import glob
 import pandas
 for filename in glob.glob('data/*.csv'):
    contents = pandas.read_csv(filename)
-   if len(contents) < 85:
+   if len(contents) < 50:
```

Comment:
85 is necessary for code to run.  All files are around 84-87 records in length.  Should change the header 50 to match insead.